skycatalogs/catalog_creator.py

```diff
@@ -237,7 +238,8 @@ def __init__(self, parts, area_partition=None, skycatalog_root=None,
                  galaxy_stride=1000000, provenance=None,
                  dc2=False, sn_object_type='sncosmo', galaxy_type='cosmodc2',
                  include_roman_flux=False, star_input_fmt='sqlite',
-                 sso_truth=None, sso_sed=None, sso_partition='healpixel'):
+                 sso_truth=None, sso_sed=None, sso_partition='healpixel',
+                 run_options=None):
```

Comment:
Can you add a docstring entry for `run_options` please?

Reply:
Yes, certainly.  Just an oversight.